sound/hda/intel-dsp-config.c

```diff
@@ -472,6 +472,14 @@ static const struct config_entry config_table[] = {
 	},
 #endif
 
+/* Meteor Lake */
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_METEORLAKE)
+	{
+		.flags = FLAG_SOF,
```

Comment:
Shouldn't this be:
               .flags = FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE,

Reply:
> Shouldn't this be: .flags = FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE,

I am not sure about it. Updated to `FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE`